### PR TITLE
Add "Launch in Deepnote" button

### PR DESCRIPTION
**Pull request by danzvara**
Hi Sebastian, I'm Dan from Deepnote. We're a small startup creating a better data science notebook compatible with Jupyter and one of the things we're really interested is making it easier for people to try and learn from existing projects. I was trying out Deepnote with some interesting public repos, including your and since I already set it up for myself, I thought it might be useful for others too to have a one-click button to run your repo, so I submitted a PR. 
Hope you find it useful!

Thanks for your work,
Dan

### CODE REVIEW

**Comment by rasbt** [comment]
Hi Dan,

this looks potentially useful to me! However, when I click on the button, I get an empty view prompting me to add files (see below). Is this intended?
Screen Shot 2020-10-19 at 10 05 04 AM

**Comment by danzvara** [comment]
That's an unlucky setup with cloned repo's, you need to click on the file icon in the sidebar to see the files in the repo :) I will make sure it's fixed asap!

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 <hr>
 **Tutorials, examples, collections, and everything else that falls into the categories: pattern classification, machine learning, and data mining.**
 <br>
+<a href="https://deepnote.com/launch?template=pytorch&url=https%3A%2F%2Fgithub.com%2Frasbt%2Fpattern_classification"><img src="https://deepnote.com/buttons/launch-in-deepnote-small.svg" /></a>
```
Comment (danzvara) [inline README.md:11]:
```suggestion
<a href="https%3A%2F%2Fgithub.com%2Frasbt%2Fpattern_classification%2Fblob%2Fmaster%2Fmachine_learning%2Fscikit-learn%2Fscikit_linear_classification.ipynb"><img src="https://deepnote.com/buttons/launch-in-deepnote-small.svg" /></a>
```

**Comment by danzvara** [comment]
@rasbt fixed, the link will open one of the notebooks now :)

**Pull request closed without merging**